### PR TITLE
BAP-486 - Add Issues Collection Task

### PR DESCRIPTION
**Pull request by RobiNino**

### CODE REVIEW

File: src/main/resources/atlassian-plugin.xml [hunk 1]
```diff
@@ -149,6 +149,20 @@
         <exporter class="org.jfrog.bamboo.exporter.ArtifactoryTaskExporter"/>
     </taskType>
 
+    <!--Collect issues task-->
+    <taskType key="artifactoryCollectIssuesTask" name="Artifactory Collect Issues"
+              class="org.jfrog.bamboo.task.ArtifactoryCollectIssuesTask">
+        <description>Artifactory Collect Issues Task</description>
```
Comment (eyalbe4) [inline src/main/resources/atlassian-plugin.xml]: I suggest renaming the task name to:
**Artifactory Build Issues Task**
If we're going for this change, let's also rename all the relevant methods and variables from ...collectIssues... to ...buildIssues...

File: src/main/resources/i18n-jfrog.properties [hunk 2]
```diff
@@ -312,6 +312,15 @@ artifactory.task.publishBuildInfo.header.username.description = Override the def
 artifactory.task.publishbuildinfo.header.password = Override Default Password
 artifactory.task.publishbuildinfo.header.password.description = The password of the user entered above.
 
+#Artifactory Collect Issues
+artifactory.task.collectIssues.title = Artifactory Collect Issues
```
Comment (eyalbe4) [inline src/main/resources/i18n-jfrog.properties]: I suggest renaming *Artifactory Collect Issues* to *Artifactory Build Issues*.

File: src/main/resources/i18n-jfrog.properties [hunk 3]
```diff
@@ -312,6 +312,15 @@ artifactory.task.publishBuildInfo.header.username.description = Override the def
...
+artifactory.task.collectIssues.configFilePath = Config File Path
+artifactory.task.collectIssues.configJson = Config Json
```
Comment (eyalbe4) [inline src/main/resources/i18n-jfrog.properties]: I suggest changing to:
**artifactory.task.collectIssues.config.source = Config Source**
to be compatible with "Spec Source".

File: src/main/resources/i18n-jfrog.properties [hunk 4]
```diff
@@ -312,6 +312,15 @@ artifactory.task.publishBuildInfo.header.username.description = Override the def
...
+artifactory.task.collectIssues.header.username = Override Default Username
+artifactory.task.collectIssues.header.username.description = Override the default username used for fetching previous build info to Artifactory.
```
Comment (eyalbe4) [inline src/main/resources/i18n-jfrog.properties]: I suggest rephrasing the description to:
*Override the default username used for reading the information of previous build runs from Artifactory.*

File: src/main/resources/i18n-jfrog.properties (hunk 2 above)
Comment (RobiNino) [reply src/main/resources/i18n-jfrog.properties]: Changed to Artifactory Collect Build Issues, as we discussed

**Pull request merged**